src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThroughFallThroughLotsOfComment.java

```diff
@@ -0,0 +1,72 @@
+/*
+FallThrough
+checkLastCaseGroup = (default)false
+reliefPattern = (default)falls?[ -]?thr(u|ough)
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.fallthrough;
+
+public class InputFallThroughFallThroughLotsOfComment {
+
+    void method(int lineNumber) {
+        switch (lineNumber) {
+            case 3:
+                Object pi = null;
+                int line = 2;
+                // fall-through intended - the line contains already the first entries
+                // the following 4 lines contain additional satellite ids
+            case 4:
+            case 5:
+            case 6:
+            case 7: {
+                int lineLength =  2;
+                int startIdx = 9;
+                while (lineLength++ < 2&& (startIdx + 3) <= lineLength) {
+                    String satId;
+                    startIdx += 3;
+                }
+                break;
+            }
+
+            case 14:
+                // ignore additional custom fields
+                break;
+
+            // load base numbers for the standard deviations of
+            // position/velocity/clock components
+            case 15: {
+                // String base = line.substring(3, 13).trim();
+                // if (!base.equals("0.0000000")) {
+                //    // (mm or 10**-4 mm/sec)
+                //    pi.posVelBase = Double.valueOf(base);
```

Comment:
Remove extra comments that don't contribute anything meaningful. Just keep 1-2 lines only

Reply:
done